CONTRIBUTING.md

```diff
@@ -0,0 +1,161 @@
+
+# Contributing to Jackson
+
+Thank you for your interest in contributing to Jackson! Your contributions are essential in making Jackson even better. Whether you're fixing a bug, implementing new features, or suggesting improvements, your involvement is valued and appreicated.
+
+
+## Code Style
+
+Please follow the [Node Style Guide](https://github.com/felixge/node-style-guide).
+
+## Getting Started
+
+### 1. Fork the Repository
+
+Begin by forking the Jackson repository. This creates a copy of the project where you can make and test your changes without impacting the original project.
+
+[Fork this repository](https://github.com/boxyhq/jackson/fork)
+
+
+### 2. Clone the Repository
+
+Clone your forked repository to your local development environment using this command:
+
+```shell
+git clone https://github.com/your-username/jackson.git
+```
+
+### 3. Set Up
+
+Navigate to the project folder and install necessary dependencies:
+
+```shell
+cd jackson
+```
+
+#### Install custom dependencies
+
+``` shell
+npm run custom-install
+```
+
+#### Add Environment Variables
+
+```shell
+cp .env.example .env
+npm run dev
+```
+
+Please update .env with your values. See the complete list of [Environment Variables](https://boxyhq.com/docs/jackson/deploy/env-variables).
+
+### 4. Build and Run
+
+Ensure the project is ready for development:
+
+```shell
+npm run build
+npm run start
+```
+
+Visit [http://localhost:5225](http://localhost:5225) in your browser. If you see a sign-in page, you're on the Admin Portal.
+
+Read the full documentation [here](https://boxyhq.com/docs/jackson/deploy/).
+
+## Contribution  
+
+
+
+### Create a new branch 
+
+```shell
+git checkout -b your-branch-name
+```
+#### or
+
+```shell
+
+git branch your-branch-name
+```
+### Stage your changes
+
+``` shell
+git add your-file-name
+```
+
+or simply stage everything,
+
+``` shell
+git add .
+```
+
+### Make and commit your changes with a note
+
+```shell
+git commit -m ""enter the message for changes that need to be committed"
```

Comment:
git commit -m "enter the message for changes that need to be committed"

Reply:
Sorry for such typos i have corrected all the typos and logical errors,thanks @deepakprabhakara